src/main/resources/messages.properties

```diff
@@ -936,10 +936,41 @@ doc-upload.expired=The time window to upload documents has run out.
 doc-upload.finalized=You have already uploaded documents
 
 # Email
-email-to-recipient.subject=Application Received
-email-to-recipient.body=<html><body><p>We received your Maryland Benefits application. Because you applied for food and/or cash programs, \
-  you are automatically considered for an energy benefit (OHEP).</p><p>Application status: In review</p><p>Your county will contact you \
-  via mailing address and/or phone number with next steps.</p><p>If you want an update on your case, please call your county.</p></body></html>
+email-to-recipient.subject=Thank you for applying on MarylandBenefits.org
+email-to-recipient.body=<html><body> \
+  <p>We received your Maryland Benefits application.</p> \
+  <p> \
+    <strong>Confirmation Number:</strong>\
+    #{0} \
+    <br/>\
+    <strong>Application Status:</strong>\
+    In review \
+    <br/>\
+    <strong>Next Steps:</strong>\
+    Your county will contact you via email, mail or phone with next steps. \
+  </p> \
+  <p> \
+    If you want an update on your case, please call \
+    <br/>\
+    Queen Anne's County: 401-758-8000 \
+    <br/>\
+    Baltimore County: 1-800-332-6347 \
+  </p> \
+  <p> \
+    <strong>Sending additional documents:</strong>\
+    If you would like to email additional documents, you can email Baltimore County at baltimorecounty.dmc@maryland.gov  or Queen Anne's County at qacfia.customeraccount@maryland.gov. \
```

Comment:
The apostrophe in Queen Anne's county is not showing up in the email for some reason.  It might be an encoding issue we have sometimes when we copy content from different places (especially figma). If you delete and re-add it, it might solve the issue.

Reply:
Done? How did you verify that?